Cargo.toml

```diff
@@ -134,3 +134,6 @@ bulletproofs = { git = "https://github.com/eranrund/bulletproofs", rev = "e8e8ef
 
 # Patched to disable the cpuid instruction because that is incompatible with our sgx builds.
 cpuid-bool = { git = "https://github.com/eranrund/RustCrypto-utils", rev = "e00f5f16e395349c2c2395aac9287da0ac843a73" }
+
+# We need to patch aes-gcm so we can make some fields/functions/structs pub in order to have a constant time decrypt
+aes-gcm = { git = "https://github.com/xoloki/AEADs", rev = "d1a8517d3dd867ed9c5794002add67992a42f6aa" }
```

Comment:
Any idea what are the chances of this being accepted into upstream?

Reply:
Basically zero.  Chris and I both talked with him for a while, but he was only willing to make some very minor changes, which were not sufficient to our needs.  His final position was that it was "only like 80 lines of code, so write your own version that does exactly what you want".  Which seemed reasonable.

Chris wants to do a more detailed rewrite anyway, so we shouldn't worry too much about taking on patch for now.